app/frontend/src/components/file-row.js

```diff
@@ -8,14 +8,22 @@ import moment from 'moment'
 import { getClassNameForFilename } from '../utils/files'
 import { SelectableRow } from './selectable-row'
 import { EthAddress } from './eth-address'
+import { Label } from './label'
 
 fontawesome.library.add(solid.faDownload)
 
-export const FileRow = ({ file, onClick, onDownloadClick, selected }) =>
+export const FileRow = ({ file, onClick, onLabelClick, onDownloadClick, selected }) =>
   <Container {...{ onClick, selected }}>
     <NameCell>
       <Name>
-        <FontAwesomeIcon icon={getClassNameForFilename(file.name)} /> {file.name}
+        <FontAwesomeIcon icon={getClassNameForFilename(file.name)} />
+        <FileName>{file.name}</FileName>
+        {file.labels.map(label =>
```

Comment:
Created this [issue](https://github.com/espresso-org/aragon-drive/issues/74) so we can work on this at a later time since it is probably a rather uncommon scenario.